### PR TITLE
 run copier update, bring the bots in, run tox -e deps

### PR DESCRIPTION
**Pull request by MridulS**
This also updates the deps, which should fix the failing nightly tests of esspolarization.

The last tox -e deps was probably run in a python 3.10 env while the CI wants py3.9 (related https://github.com/scipp/copier_template/issues/163)

### CODE REVIEW

File: .copier-answers.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Changes here will be overwritten by Copier; NEVER EDIT MANUALLY
-_commit: 8cfdb1b
+_commit: 6ee5264
 _src_path: gh:scipp/copier_template
 description: Polarization data reduction for the European Spallation Source
 max_python: '3.11'
```
Comment (SimonHeybrock) [inline .copier-answers.yml:5]: I think we can drop Python 3.9 and max can probably be 3.12.

**Review by SimonHeybrock — APPROVED** [review]

**Comment by MridulS** [comment]
Hmm, has anyone seen this before? Bumping up `python-version-ci` ends up with this weird unrunable required test `Tests (ubuntu-22.04, 3.9, py39) / test` in this case.

**Comment by YooSunYoung** [comment]
> Hmm, has anyone seen this before? Bumping up `python-version-ci` ends up with this weird unrunable required test `Tests (ubuntu-22.04, 3.9, py39) / test` in this case.

You mean the CI tests status? It's because of the main branch protection rule. We have to update them manually.

**Comment by YooSunYoung** [comment]
> > Hmm, has anyone seen this before? Bumping up `python-version-ci` ends up with this weird unrunable required test `Tests (ubuntu-22.04, 3.9, py39) / test` in this case.
> 
> You mean the CI tests status? It's because of the main branch protection rule. We have to update them manually.

@scipp/ess-maintainers I don't have the permit to update the main branch protection rule.
Can someone remove `Tests (ubuntu-22.04, 3.9, py39) / test` from the required tests and add `CI / Tests (ubuntu-22.04, 3.10, py310) / test (pull_request) ` instead? Thanks!

**Pull request merged**